### ISSUE TITLE
[REQ]: Acrossthetasman

### ISSUE THREAD

**Issue by antigravity83** [Jackett/Jackett#1628]
id: acrossthetasman
name: Across The Tasman
description: "ATT is a Private torrent site for Rugby and other sports played in Australia"
type: private
links:
  - https://acrossthetasman.com/

**Comment by ghost** [comment]
send an invite to jackettapp at gmail and i'll add it

**Comment by antigravity83** [comment]
Hi there

I don't have any invites available unfortunately but can share my login so you can do your thing?

**Comment by garfield69** [comment]
whats the address for this site please?

**Comment by antigravity83** [comment]
https://acrossthetasman.com/

**Comment by garfield69** [comment]
thanks.
registrations currently closed. invite needed.

**Comment by antigravity83** [comment]
Don't have any invites unfortunately I never use it because it's not in Jackett haha

**Comment by helenclarko** [comment]
I don't believe AcrosstheTasman has a feed, it's a forum, I'm not sure how the searching will work.

They also don't do invites anymore, only staff can invite/recruit people.

What are the options from here?
Can I send page source or something?

**Comment by ilike2burnthing** [comment]
[v0.17.513](https://github.com/Jackett/Jackett/releases/tag/v0.17.513)

**Comment by garfield69** [comment]
https://acrossthetasman.com/ > TCP Connection Failed (Timed out)

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://acrossthetasman.com/ > TCP Connection Failed (Timed out)

**Comment by ilike2burnthing** [comment]
back up
